packages/documentation/copy/en/reference/Type Compatibility.md

```diff
@@ -70,6 +70,14 @@ greet(dog); // OK
 Note that `dog` has an extra `owner` property, but this does not create an error.
 Only members of the target type (`Pet` in this case) are considered when checking for compatibility.
 
+Be aware, however, that [object literals](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#other-important-typescript-types)
```

Comment:
Er, what we should really have is a section on excess property checks... we used to have one in the old handbook, but apparently there is no current alternative? https://www.typescriptlang.org/docs/handbook/interfaces.html#excess-property-checks

Reply:
Done. Please see the latest commit.